src/scripts.js

```diff
@@ -271,11 +278,6 @@ function filterNonSearched(filtered) {
   hideUnselectedRecipes(found);
 }
 
-function createRecipeObject(recipes) {
-  recipes = recipes.map(recipe => new Recipe(recipe));
-  return recipes
-}
-
```

Comment:
this was moved into the file of a recipe I think we might need to revisit that idea